app/livechat/server/api/lib/livechat.js

```diff
@@ -141,3 +142,8 @@ export function settings() {
 		},
 	};
 }
+
+export async function getExtraConfigInfo(room, callback) {
+	const extraConfig = await callbacks.run('livechat.onLoadConfigApi', room);
```

Comment:
Why callback and promise? Can we keep it with promise only?

Reply:
Sure!
Done.